### ISSUE TITLE
BingX dependency alignment in CryptoClients.Net

### ISSUE THREAD

**Issue by PhilAlain** [JKorf/CryptoClients.Net#5]
Hello Jan,

I have the following code:
                    services.AddCryptoClients(globalOptions =>
                    {
                        globalOptions.OutputOriginalData = true;
                        globalOptions.ApiCredentials = new CryptoClients.Net.Models.ExchangeCredentials
                        {
                        };
                    });

And when starting it:
Unhandled exception. System.MissingMethodException: Method not found: '!!0 CryptoExchange.Net.Objects.Options.RestExchangeOptions`1.Copy()'.
   at BingX.Net.Objects.Options.BingXRestOptions.Copy()
   at Microsoft.Extensions.DependencyInjection.ServiceCollectionExtensions.AddBingX(IServiceCollection services, Action`1 defaultRestOptionsDelegate, Action`1 defaultSocketOptionsDelegate, Nullable`1 socketClientLifeTime)
   at Microsoft.Extensions.DependencyInjection.ServiceCollectionExtensions.AddCryptoClients(IServiceCollection services, Action`1 globalOptions, Action`1 binanceRestOptions, Action`1 binanceSocketOptions, Action`1 bingxRestOptions, Action`1 bingxSocketOptions, Action`1 bitfinexRestOptions, Action`1 bitfinexSocketOptions, Action`1 bitgetRestOptions, Action`1 bitgetSocketOptions, Action`1 bitMartRestOptions, Action`1 bitMartSocketOptions, Action`1 bybitRestOptions, Action`1 bybitSocketOptions, Action`1 coinbaseRestOptions, Action`1 coinbaseSocketOptions, Action`1 coinExRestOptions, Action`1 coinExSocketOptions, Action`1 coinGeckoRestOptions, Action`1 cryptoComRestOptions, Action`1 cryptoComSocketOptions, Action`1 gateIoRestOptions, Action`1 gateIoSocketOptions, Action`1 htxRestOptions, Action`1 htxSocketOptions, Action`1 krakenRestOptions, Action`1 krakenSocketOptions, Action`1 kucoinRestOptions, Action`1 kucoinSocketOptions, Action`1 mexcRestOptions, Action`1 mexcSocketOptions, Action`1 okxRestOptions, Action`1 okxSocketOptions, Action`1 whiteBitRestOptions, Action`1 whiteBitSocketOptions, Nullable`1 socketClientLifetime)

Since we do not use BingX in our project, this is probably a dependency alignment issue.

Always impressed by your work !

Best Regards,
Phil

**Comment by JKorf** [comment]
Hi, this is probably because you have the CryptoClients.Net package installed, but also individual exchange implementations. Since CryptoClients.Net internally already references the other packages only that package is needed.
You now get an error because the latest CryptoClients.Net package and one or more of the exchange packages aren't using the same CryptoExchange.Net base package internally.

In short, you can remove other package references and just keep the CryptoClients.Net package and it should work correctly.

**Comment by PhilAlain** [comment]
Corrected with the latest release :-)
